Source/Fuse.Common/Tests/FuseTest/TestBase.uno

```diff
@@ -62,7 +62,7 @@ namespace FuseTest
 			Diagnostics.DiagnosticReported -= OnDiagnostic;
 
 			if (_diagnosticsQueue.Count > 0)
-				throw new Exception("Unchecked, queued Diagnostics!");
+				throw new Exception("Unchecked, queued Diagnostics! Count="  + _diagnosticsQueue.Count);
```

Comment:
Do we really want to do this? Wouldn't it be better to create a new exception-type that includes the queued diagnostic list?

Reply:
This is just debug information that is useful when debugging test cases (nothing is meant to read this programmatically). Since it's solely used in tests I think the odd message is okay.

Reply:
Fair enough.